src/core/lib/iomgr/lockfree_event.cc

```diff
@@ -89,7 +89,11 @@ void LockfreeEvent::DestroyEvent() {
 
 void LockfreeEvent::NotifyOn(grpc_closure* closure) {
   while (true) {
-    gpr_atm curr = gpr_atm_no_barrier_load(&state_);
+    /* This load needs to be an acquire load because this can be a shutdown
+     * error that we might need to reference. Adding acquire semantics makes
+     * sure that the shutdown error has been inited properly before us
```

Comment:
s/inited/initiated

Reply:
initialized yes. me being lazy. will correct it :)